assignments/lambda-classes.js

```diff
@@ -1 +1,118 @@
 // CODE here for your Lambda Classes
+
+class Person {
+  constructor(props){
+    this.name = props.name;
+    this.age = props.age;
+    this.location = props.location;
+    this.gender = props.gender;
+  }
+  speak() {
+    // This method logs out a phrase Hello my name is Fred, I am from Bedrock where name and location are the object's own props
+    return console.log(`Hello my name is ${this.name}, I am from ${this.location}.`)
+  }
+}
+
+class Instructor extends Person {
+  constructor(props){
+    super(props);
```

Comment:
Good job just using `props` in the constructor! This is what you'll see most often (versus more detailed names for the parameter).